src/main/java/amazonreviewsapp/springboot/model/User.java

```diff
@@ -0,0 +1,74 @@
+package amazonreviewsapp.springboot.model;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import java.util.Set;
+
+@Entity
+@Table(name = "users")
+public class User {
```

Comment:
Is this the user from reviews or that the one that will be using your app?

Reply:
Each user entity has its own password which is a duplicate of their profile name, will test that later when will add a security layer